torchtext/data/datasets_utils.py

```diff
@@ -165,7 +165,8 @@ def download_extract_validate(root, url, url_md5, downloaded_file, extracted_fil
     dataset_tar = download_from_url(url, path=os.path.join(root, downloaded_file),
                                     hash_value=url_md5, hash_type=hash_type)
     extracted_files = extract_archive(dataset_tar)
-    assert extracted_file == find_match(extracted_file, extracted_files)
```

Comment:
Follow-up: We probably still want this (since this assert has helped us in the past), but we can do it as a follow-up.

Reply:
Added a follow-up task